packages/containers/src/List/selector.js

```diff
@@ -41,91 +41,74 @@ export function configureGetPagination(state, { collectionId }) {
 	return null;
 }
 
-function getCollectionData(collectionId) {
-	return state => getCollectionItems(state, collectionId);
-}
-
-function getComponentState(collectionId) {
+export function getComponentState(collectionId) {
 	return state => state.cmf.components.getIn(['Container(List)', collectionId || 'default']);
 }
 
-export function configureGetFilteredItems(configure) {
-	const localConfig = configure;
-
-	const getFilteredList = createSelector(
-		[getCollectionData(localConfig.collectionId), getComponentState(localConfig.collectionId)],
-		(items, componentState) => {
-			let results = items || localConfig.items;
-			if (componentState) {
-				const searchQuery = componentState.get('searchQuery');
-				if (searchQuery && results) {
-					results = results.filter(item => contains(item, searchQuery, localConfig.columns));
+export function sortList(componentState, items, localConfig) {
```

Comment:
We'll need to do something with this function, it deals with too many things (too complex). And should be elsewhere because it's not a redux/cmf state selector.